sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/ConversationAnalysisClientLiveTests.cs

```diff
@@ -23,30 +23,30 @@ public async Task AnalyzeConversation()
         {
             var data = new
             {
-                analysisInput = new
+                AnalysisInput = new
                 {
-                    conversationItem = new
+                    ConversationItem = new
                     {
-                        text = "Send an email to Carol about the tomorrow's demo",
-                        id = "1",
-                        participantId = "1",
+                        Text = "Send an email to Carol about the tomorrow's demo",
+                        Id = "1",
+                        ParticipantId = "1",
                     }
                 },
-                parameters = new
+                Parameters = new
                 {
-                    projectName = TestEnvironment.ProjectName,
-                    deploymentName = TestEnvironment.DeploymentName,
+                    ProjectName = TestEnvironment.ProjectName,
+                    DeploymentName = TestEnvironment.DeploymentName,
                 },
-                kind = "Conversation",
+                Kind = "Conversation",
             };
 
-            Response response = await Client.AnalyzeConversationAsync(RequestContent.Create(data));
+            Response response = await Client.AnalyzeConversationAsync(RequestContent.Create(data, PropertyNamingConvention.CamelCase));
```

Comment:
@heaths, we can revert this before merging the PR, but I wanted to keep it in this version in order to illustrate intended usage of the enum value across multiple domains.

Reply:
Feel free to keep it like it is if that helps illustrate the concept and/or garner feedback.

Reply:
TY!